specification/azsadmin/resource-manager/update/Microsoft.Update.Admin/preview/2021-07-01/Updates.json

```diff
@@ -0,0 +1,461 @@
+          "type": "string"
+        },
+        "packagePath": {
+          "description": "Path to the update package.",
+          "type": "string"
+        },
+        "packageSizeInMb": {
+          "description": "Size of update package.",
+          "type": "integer",
+          "format": "int64"
+        },
+        "publisher": {
+          "description": "Publisher of the update.",
+          "type": "string"
+        },
+        "prerequisites":{
+          "description": "List of prerequisites.",
+          "type": "array",
+		  "items": {
+            "$ref": "#/definitions/Prerequisite"
+          }
+        },
+        "updateStateProperties":{
+          "description": "Properties when update is in a specific state.",
+          "$ref": "#/definitions/UpdateStateProperties"
+        },
+        "azsUpdateProperties":{
+          "description": "Properties specific to Microsoft Azure update.",
+          "$ref": "#/definitions/AzsUpdateProperties"
+        },
+        "oemUpdateProperties":{
+          "description": "Properties specific to OEM update.",
+          "$ref": "#/definitions/OemUpdateProperties"
+        }
+      }
+    },
+    "UpdateList": {
+      "description": "List of updates.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "List of updates.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Update"
+          }
+        },
+        "nextLink": {
+          "description": "URI to the next page of updates.",
+          "type": "string"
+        }
+      }
+    },
+    "UpdateState": {
+      "description": "State of the update.",
+      "type": "string",
+      "enum": [
+        "Available",
+        "Downloading",
+        "DownloadFailed",
+        "HealthChecking",
+        "HealthCheckFailed",
+        "Invalid",
+        "HasPrerequisite",
+        "Recalled",
+        "Obsolete",
+        "ReadyToInstall",
+        "NotApplicableBecauseAnotherUpdateIsInProgress",
+        "Installing",
+        "Installed",
+        "AttentionNeeded",
+        "InstallationFailed"
+      ],
+      "x-ms-enum": {
+        "name": "UpdateState",
+        "modelAsString": true
+      }
+    },
+    "AvailabilityType": {
+      "description": "Availability type of update.",
+      "type": "string",
+      "enum": [
+        "online",
+        "local",
+        "notify"
+      ],
+      "x-ms-enum": {
+        "name": "AvailabilityType",
+        "modelAsString": true
+      }
+    },
+    "Prerequisite": {
+      "description": "Prerequisite of the update.",
+      "type": "object",
+      "properties": {
+        "title": {
+          "description": "Title of the prerequisite.",
+          "type": "string"
+        },
+        "description": {
+          "description": "Description of the prerequisite.",
+          "type": "string"
+        },
+        "infomationLink": {
+          "description": "Supported link of the prerequisite.",
```

Comment:
I instead suggest the following:
"Link providing details about prerequisite."

The difference is subtle, but I don't expect us to have a link to explaining the prerequisite.  For example, there won't be a link to a page that says "You need to install X before Y because of Z".  Instead you will just get a link that talks about the prerequisite package X (e.g. the original release notes for X and download info for the X) and it might not say anything about the thing that had the prerequisite (Y) or the reason for it (Z).